concent_api/concent_api/settings/base.py

```diff
@@ -281,6 +281,13 @@
 # Stored in a 'bytes' array, e.g. b'\xf3\x97\x19\xcdX\xda...'
 # CONCENT_ETHEREUM_PRIVATE_KEY = ''
 
+# A global constant defining the URL of the storage server
+STORAGE_SERVER_INTERNAL_ADDRESS = ''
```

Comment:
This should not have a default value. Even a blank one. It should just fail if no one bothers to define it.

Reply:
:+1: done